libs/wingc/src/lib.rs

```diff
@@ -193,18 +193,15 @@ pub fn type_check(scope: &mut Scope, types: &mut Types) -> Diagnostics {
 
 // TODO: refactor this (why is scope needed?) (move to separate module?)
 fn add_builtin(name: &str, typ: Type, scope: &mut Scope, types: &mut Types) {
-	let sym = Symbol {
-		name: name.to_string(),
-		span: WingSpan::global(),
-	};
+	let sym = Symbol::global(name);
 	scope
 		.env
 		.borrow_mut()
 		.as_mut()
 		.unwrap()
 		.define(
 			&sym,
-			SymbolKind::make_variable(types.add_type(typ), false),
+			SymbolKind::make_variable(types.add_type(typ), false, Phase::Independent),
```

Comment:
Do all our builtin functions have to be Phase::Independent?
I know the current 4 ones are, but maybe it makes sense to pass the phase as a parameter (as an anecdote @MarkMcCulloh provided a suggestion for an inflight builtin [here](https://winglang.slack.com/archives/C048QCN2XLJ/p1675353044361399)).

Reply:
I think this should be fine as-is, the symbol itself can be phase independent while it represents an inflight function (my proposal).

Reply:
Actually, @MarkMcCulloh, the phase of the symbol is deduced from the phase of the `SymbolEnv` it's defined in (in this case the global scope is preflight). The argument passed to `make_variable` is the phase of the function itself (in this case hard-coded phase independent).
@staycoolcall911 I think that when we even have an inflight builtin we can easily change this. I'm not sure Mark's `includejs` is necessarily inflight only...